CHANGES.md

```diff
@@ -15,6 +15,7 @@ Last release of this project was 18th of March 2024.
   - fix(ck4): Place formula in the correct position after inserting italics (or bold or underlined) text in a new line. #KB-43453
   - feat: Upgrade DOMPurify library to avoid deletion of some MathML tags by error. #KB-44328
   - feat: Add new 'Commit hash' field to html-viewer demo. #KB-44411
+  - fix(viewer): Render the formula that is not been cached. #KB-44942
```

Comment:
I don´t really like this changelog message. Actually, it doesn´t match the one provided in the Kanbanize card, which by the way I don´t like either. I suggest:

```suggestion
  - fix(viewer): Avoid formulas not rendering after some of them contains invalid MathML. #KB-44942
```